test/specs/common/tests/02.column.js

```diff
@@ -0,0 +1,194 @@
+                    });
+                });
+
+                describe('should call printBoolean() to format,', function() {
+                    it('boolean columns correctly.', function() {
+                        var spy = spyOn(formatUtils, 'printBoolean').and.callThrough();
+                        var col = table1_schema2.columns.getByPosition(5);
+
+                        var trueBoolean = col.formatvalue(true);
+                        expect(spy).toHaveBeenCalledWith(true, undefined);
+                        expect(trueBoolean).toEqual(jasmine.any(String));
+                        expect(trueBoolean).toBe('true');
+
+                        var falseBoolean = col.formatvalue(false);
+                        expect(spy).toHaveBeenCalledWith(false, undefined);
+                        expect(falseBoolean).toEqual(jasmine.any(String));
+                        expect(falseBoolean).toBe('false');
+
+                        expect(spy.calls.count()).toBe(2);
+                    });
+                });
+
+                describe('should call printTimestamp() to format,', function() {
+                    it('timestamptz columns correctly.', function() {
+                        var spy = spyOn(formatUtils, 'printTimestamp').and.callThrough();
+                        var testVal = '2011-05-06T08:25:25-07:00';
+                        var col = table1_schema2.columns.getByPosition(6);
+                        var formattedValue = col.formatvalue(testVal);
+                        expect(spy).toHaveBeenCalledWith(testVal, undefined);
+                        expect(formattedValue).toEqual(jasmine.any(String));
+                        expect(formattedValue).toBe(new Date(testVal).toLocaleString());
+                    });
+                });
+
+                describe('should call printFloat() to format,', function() {
+                    var formattedValue = undefined;
+
+                    it('float4 columns correctly.', function() {
+                        var spy = spyOn(formatUtils, 'printFloat').and.callThrough();
+                        var col = table1_schema2.columns.getByPosition(7);
+                        formattedValue = col.formatvalue(11.1);
+                        expect(spy).toHaveBeenCalledWith(11.1, undefined);
+                        expect(formattedValue).toBe('11.10');
+                    });
+
+                    it('float8 columns correctly.', function() {
+                        var spy = spyOn(formatUtils, 'printFloat').and.callThrough();
+                        var col = table1_schema2.columns.getByPosition(8);
+                        var options = {numDecDigits: 7};
+                        formattedValue = col.formatvalue(234523523.023045230450245, options);
+                        expect(spy).toHaveBeenCalledWith(234523523.023045230450245, options);
+                        expect(formattedValue).toBe('234,523,523.0230452');
+                    });
+
```

Comment:
Should also cover a case where both command and decimal will show up (e.g. a float > 1000)